IdentityCore/src/webview/embeddedWebview/challangeHandlers/ios/MSIDCertAuthHandler.m

```diff
@@ -126,6 +126,7 @@ + (BOOL)handleChallenge:(NSURLAuthenticationChallenge *)challenge
         NSMutableDictionary *newQueryItems = [NSMutableDictionary new];
         NSString *redirectSchemePrefix = [NSString stringWithFormat:@"%@://", s_redirectScheme];
         
+        newQueryItems[MSID_BROKER_IS_PERFORMING_CBA] = @"true";
```

Comment:
Is it hard coded all time?

Reply:
That's a good point, I do see throughout IDCore that @"true" is used frequently, so it would make sense to create a constant for consistency and a single point for change.. I'll investigate